scripts/delete-data.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
this can probably go, not sure this helps anyone outside of our infrastructure.

Reply:
Deleted.